crates/rpc/rpc/src/debug.rs

```diff
@@ -390,9 +391,21 @@ where
                         return Ok(frame)
                     }
                     GethDebugBuiltInTracerType::FlatCallTracer => {
-                        return Err(
-                            EthApiError::Unsupported("Flatcall tracer is not supported yet").into()
-                        )
+                        let flat_call_config = tracer_config
+                            .into_flat_call_config()
+                            .map_err(|_| EthApiError::InvalidTracerConfig)?;
+                        
+                        let mut inspector = TracingInspector::new(
+                            TracingInspectorConfig::from_flat_call_config(&flat_call_config)
+                        );
+                        
+                        // Now `res` is the unwrapped value, not an Option
+                        let res = None;
+
```

Comment:
this is a good start, we need to replicate what we do here:

https://github.com/paradigmxyz/reth/blob/242a3ab06befbd06a51cad1602ca80df6930a4cb/crates/rpc/rpc/src/trace.rs#L342-L358

and
https://github.com/paradigmxyz/reth/blob/242a3ab06befbd06a51cad1602ca80df6930a4cb/crates/rpc/rpc/src/trace.rs#L366-L374
I think here we can ignore the block reward stuff

Reply:
tx_info should just be default here right since it isn't actually tx and is just a call? 

(quickly update code to use default, lmk if there is something else i should be fetching)